dbt_subprojects/hourly_spellbook/models/_sector/blobs/ethereum/_schema.yml

```diff
@@ -85,9 +85,7 @@ models:
         description: "hash of the L1 transaction"
       - *beacon_slot_number
       - *beacon_epoch
-      - name: tx_from
-        description: "address of the sender"
-      - name: tx_to
```

Comment:
Removal due to neither `tx_from` nor `tx_to` being present in the final selection.